src/Dash.jl

```diff
@@ -112,8 +112,4 @@ function __init__()
 end
 
 
-JSON3.StructTypes.StructType(::Type{DashBase.Component}) = JSON3.StructTypes.Struct()
-JSON3.StructTypes.excludes(::Type{DashBase.Component}) = (:name, :available_props, :wildcard_regex)
-
-
```

Comment:
FYI this piece of logic in now [here](https://github.com/sjkelly/DashBase.jl/blob/74ef396cb54297d6c7130235993773bfd0ccf668/src/components.jl#L58-L59) in DashBase.jl - which is a good thing to avoid type-piracy. 

since https://github.com/plotly/DashBase.jl/pull/2/